src/components/NavBar.tsx

```diff
@@ -41,9 +46,11 @@ function NavBar() {
   const renderedNavItems = navTitles.map((item) => renderNavItem(item))
 
   return (
-    <NavContainer>
-      <MainContainer>{renderedNavItems}</MainContainer>
-    </NavContainer>
+    <div>
```

Comment:
In the future, can wrap it like this instead fyi. See https://reactjs.org/docs/fragments.html#short-syntax.

Also, is this additional fragment necessary?

```suggestion
    <>
```

Reply:
Fixed in commit bb5b12e